src/drivers/intel/cavs/dmic.c

```diff
@@ -153,11 +158,11 @@ static uint64_t dmic_work(void *data)
 	spin_lock(&dai->lock);
 
 	/* Increment gain with logaritmic step.
-	 * Gain is Q2.30 and gain modifier is Q2.14.
+	 * Gain is Q2.30 and gain modifier is Q12.20.
```

Comment:
you may want to fix all the 'logarithmic' typos.

Reply:
Again :^) Yes...